cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/AbstractCruiseControlResponse.java

```diff
@@ -17,9 +18,16 @@
 
 public abstract class AbstractCruiseControlResponse implements CruiseControlResponse {
   protected String _cachedResponse;
+  protected KafkaCruiseControlConfig _config;
 
   public AbstractCruiseControlResponse() {
     _cachedResponse = null;
+    _config = null;
```

Comment:
Will the default constructor be relevant if we require `_config` to be passed to all response? Currently only missing one seems `BrokerStats`.